packages/smooth_app/lib/helpers/permission_helper.dart

```diff
@@ -23,6 +25,13 @@ class PermissionListener extends ValueNotifier<DevicePermission> {
   Future<void> askPermission(
     Future<bool?> Function() onRationaleNotAvailable,
   ) async {
+    // Prevent multiples calls to this method
+    if (_status == _DevicePermissionStatus.asked) {
+      return;
+    }
+
+    _status = _DevicePermissionStatus.asked;
```

Comment:
... then it would be something else here.

Reply:
Mmm I'm not sure to understand what you want to achieve exactly 🤔